test/test_runner/test_runner.cpp

```diff
@@ -150,6 +150,12 @@ bool TestRunner::checkPlanResult(std::unique_ptr<QueryResult>& result, TestState
         while (std::getline(expectedTuplesFile, line)) {
             testAnswer.expectedResult.push_back(line);
         }
+        if (testAnswer.expectedResult.size() != testAnswer.numTuples) {
+            spdlog::error("PLAN{} NOT PASSED.", planIdx);
+            spdlog::info("PLAN: \n{}", planStr);
+            spdlog::info("TUPLE COUNT NOT MATCHING.");
```

Comment:
Can u provide a bit more info here? such as expected num of tuples, and actual num of tuples. though we can find these through test file and answer csv files, but better provide some convenience when possible here